job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/step/StepContextSpec.groovy

```diff
@@ -61,6 +61,41 @@ class StepContextSpec extends Specification {
         1 * jobManagement.requireMinimumPluginVersion('ssh', '1.3')
     }
 
+    def 'call xshell method with minimum options'() {
+        when:
+        context.xshell {
+            commandLine 'echo "Hello"'
```

Comment:
the commandLine option is not required by your implementation, so this does not test the minimum options.